src/ansys/fluent/visualization/__init__.py

```diff
@@ -61,5 +62,152 @@ def version_info() -> str:
 )
 from ansys.fluent.visualization.graphics import Graphics  # noqa: F401
 from ansys.fluent.visualization.plotter import Plots  # noqa: F401
-from ansys.fluent.visualization.registrar import get_renderer, register_renderer
+from ansys.fluent.visualization.registrar import (
+    _visualizer,
```

Comment:
@prmukherj With all this code added here directly in the `__init__`, obscure names like `_visualizer` are exposed by the module.
```python
>>> import ansys.fluent.visualization as v
>>> v._visualizer
{'matplotlib': <class 'ansys.fluent.visualization.plotter.matplotlib.plotter_defns.Plotter'>, 'pyvista': <class 'ansys.fluent.visualization.plotter.pyvista.plotter_defns.Plotter'>}
```

Reply:
Created a separate config.py to resolve this.